src/lib/types.ts

```diff
@@ -13,9 +13,9 @@ export interface Session {
    * A timestamp of when the token will expire. Returned when a login is confirmed.
    */
   expires_at?: number
-  refresh_token: string
+  refresh_token?: string
   token_type: string
-  user: User
+  user: User | null
```

Comment:
We need to allow the `user` to be null

Also since we can't expect a refresh_token, then that needs to be optional